contrib/completion/bash/docker

```diff
@@ -407,6 +408,7 @@ __docker_complete_log_drivers() {
 
 __docker_complete_log_options() {
 	# see docs/reference/logging/index.md
+	local gcplogs_options="gcp-project labels env log-cmd"
```

Comment:
same here: please insert in alphabetical order, with the options alphabetically sorted.
Thanks for caring for the completions!